api/tests/conftest.py

```diff
@@ -124,6 +126,12 @@ def enable_factory_create(monkeypatch, db_session) -> db.Session:
     return db_session
 
 
+@pytest.fixture
+def test_api_schema(db_client):
+    db_config = PostgresDBConfig()
+    return f"{db_config.schema_prefix_override}{Schemas.API}"
+
+
```

Comment:
How about adding a new fixture that creates the prefix string, that both this fixture and `db_client()` depend on. In `db_client()` it would pass to `create_isolated_db()` as an argument. Suggested implementation:

```suggestion
@pytest.fixture(scope="session")
def db_schema_prefix():
    return f"test_{uuid.uuid4().int}_"


@pytest.fixture
def test_api_schema(db_schema_prefix):
    return f"{db_schema_prefix}{Schemas.API}"


```